components/provisioning/org.wso2.carbon.identity.provisioning/src/main/java/org/wso2/carbon/identity/provisioning/OutboundProvisioningManager.java

```diff
@@ -469,7 +469,8 @@ public void provision(ProvisioningEntity provisioningEntity, String serviceProvi
 
                 String[] provisionByRoleList = new String[0];
 
-                if (provisioningIdp.getProvisioningRole() != null) {
+                if (provisioningIdp.getProvisioningRole() != null && StringUtils.isNotBlank(provisioningIdp
```

Comment:
I believe isNotBlank is checking the null condition as well.

Reply:
Addressed with: https://github.com/wso2/carbon-identity-framework/pull/5408/commits/b0f45bd7027eccb8b5f2e37e7007f7da773c8bd5